src/extension.ts

```diff
@@ -830,43 +840,43 @@ export async function createFile(node: IZoweDatasetTreeNode, datasetProvider: IZ
         }
 
         // get name of data set
-        let name = await vscode.window.showInputBox({placeHolder: localize("dataset.name", "Name of Data Set")});
-        name = name.toUpperCase();
+        let name = await vscode.window.showInputBox({ placeHolder: localize("dataset.name", "Name of Data Set") });
+        if (name) {
+            name = name.toUpperCase();
```

Comment:
can you add .trim() so I will also be prevented to add datasets with spaces at the beginning.

Reply:
@jellypuno Added.